tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "ES6",
     "module": "commonjs",
-    "lib": ["dom", "es2017"],
+    "lib": ["dom", "es2019"],
```

Comment:
these changes will impact how the library is built, right? is that on purpose?

Reply:
yeah to get the latest features like flatMap so if anything, this will make the build smaller but will double-check that this will still work on all modern browsers